tests/pytests/l3_dump_test.py

```diff
@@ -264,7 +261,6 @@ def test_c_test_dump_log_entries_loc_eq_1():
                                       binary,
                                       usecase_prog_dir, 'test-main.c')
     assert unpack_rv is True
-    os.environ.pop('L3_LOC_ENABLED')
```

Comment:
This and L246 -- removes dependency on environment variables while dumping.

Dumping will figure out if LOC-encoding was active, and which scheme, by looking at the status flags recording the LOC-scheme.